src/hash.h

```diff
@@ -138,6 +139,18 @@ class CHashWriter
         return result;
     }
 
+    /** A cheap hash function that just returns 64 bits from the result, it can be
```

Comment:
I don't think this comment is still appropriate. As it samples the output, which went through a hash function, this requirement about the distribution of the input of the hash writer no longer holds.

Reply:
Good point. I also wonder about the 'cheap' part here, as it's actually just taking the prefix of the real hash, but keeping for now.